### PR TITLE
Feature/ui packaging

### PR DESCRIPTION
**Pull request by sreevatsanraman**
- [x] Packaging beta ui standalone
- [ ] Packaging rpm and deb

### CODE REVIEW

File: cdap-standalone/pom.xml [hunk 1]
```diff
@@ -148,6 +148,16 @@
         </dependency>
         <dependency>
           <groupId>co.cask.cdap</groupId>
+          <artifactId>cdap-ui</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>co.cask.cdap</groupId>
+          <artifactId>cdap-ui</artifactId>
```
Comment (albertshau) [inline cdap-standalone/pom.xml]: why do we need 2 of them?

**Comment by albertshau** [comment]
just one question. didn't examine pom changes super closely, mostly trusting its been tested and works.

File: cdap-standalone/pom.xml (hunk 1 above)
Comment (sreevatsanraman) [inline cdap-standalone/pom.xml]: Fixed.

**Comment by sreevatsanraman** [comment]
Yes. Tested and it works.

**Comment by albertshau** [comment]
lgtm

**Pull request merged**